lib/slack/handlers.ex

```diff
@@ -72,6 +72,10 @@ defmodule Slack.Handlers do
     end
   end)
 
+  def handle_slack(%{type: "im_created", channel: channel}, slack) do
```

Comment:
That's a good point, I agree with that.

I'm okay with ignoring this until someone actually needs this functionality.